src/views/document/RouteView.vue

```diff
@@ -186,6 +188,25 @@ export default {
 
       return result;
     },
+    documentType() { // is-editable mixin needs this property
+        return this.$documentUtils.getDocumentType(this.document.type);
+      },
+    PromoteHistory() {
+        const result = {};
```

Comment:
you declare result as an object but ater on it's a boolean. anyway, you don't need to declare it, just return. E.g.

```js
if (activities.includes(act)) {
  return true;
}
```